GTG/gtk/preferences.py

```diff
@@ -202,7 +180,7 @@ def on_bg_color_toggled(self, widget):
 
     def on_font_change(self, widget):
         """ Set a new font for editor """
-        self.config.set("font_name", self.fontbutton.get_font_name())
+        self.config.set("font_name", self.font_button.get_font_name())
 
 
 class ShortcutWidget(object):
```

Comment:
Idea how to implement this shortcut thing really nicely. How it should work:

The top label should contain the shortcut itself. There should be added an additional button on the right that will be used for additional settings, like in you can see in gnome-tweak on the right side (You will need that for additional settings for plugins anyway):

![tweaks](https://cloud.githubusercontent.com/assets/98588/8639855/ff7c4116-28e5-11e5-81bb-ecab8a0f45ab.png)

ASCII mockup how it could look like (vim rulezz!):
```
------------------------------------------------------------------------------------
| ---------- Add new task by pressing Ctrl+F12                        ------------ |
| | Switch |                                                          | Settings | |
| ---------- System-wide shortcut that opens a new task from anywhere ------------ |
------------------------------------------------------------------------------------
```

The string representation of shortcut widget can be obtained via the following code:

```python
        (accel_key, accel_mods) = Gtk.accelerator_parse(self.new_task_binding)
        self.show_input = Gtk.accelerator_get_label(accel_key, accel_mods)
```

When the user clicks on the settings button, a new dialog should open that says something like:
```
---------------------------------------------
|                                           |
| Press new shortcut for adding a task      |
|                                           |
|                                           |
|                                           |
|                                           |
|                                           |
---------------------------------------------
```

And then listen for all the key events. You can display the error message right in that dialog:


```
---------------------------------------------
|                                           |
| Press new shortcut for adding a task      |
|                                           |
| It's not wise to use <em>{shortcut}</em>. |
| You might have issues with typing.        |
| Try to add Control, Alt or Shift.         |
|                                           |
---------------------------------------------
```

Or just display the shortcut and wait until a proper shortcut is used. Then set the shortcut and close the dialog.

I think this would be much better than the current solution and most people either don't mind to have Ctrl+F12 as default shortcut or pressing an additional button to get the custom shortcut (or they will edit gsettings manually anyway)